meta/autoid/autoid.go

```diff
@@ -42,6 +42,8 @@ const (
 	PerformanceSchemaDBID int64 = SystemSchemaIDFlag | 10000
 	// MetricSchemaDBID is the metric_schema schema id, it's exported for test.
 	MetricSchemaDBID int64 = SystemSchemaIDFlag | 20000
+	// InspectionSchemaDBID is the inspection_schema id, it's exports for test.
+	InspectionSchemaDBID int64 = SystemSchemaIDFlag | 30000
```

Comment:
Hard allocation code here should rebase autoID? if not, what if successive user creates schema/table with the same autoID.

Reply:
All id which allocated in `SystemSchemaIDFlag` namespace will not conflict with tables created by the user.

Reply:
even if the same ID is used both in `SystemSchemaIDFlag` and common table.

Reply:
Yes, the common table ID starts from 0 and will never conflict with the system table.

Reply:
See.